docker-compose.yml

```diff
@@ -304,22 +304,13 @@ services:
   elixir:
     build:
       context: elixir
-      target: builder
+      target: compiler
```

Comment:
@jamilbk we don't need a released application here, just a compiled one - so we can use the earlier version where everything is already built. I think part of the cache was removed by mounting empty volumes, so I just removed them.